schema/data/prod/emission.sql

```diff
@@ -0,0 +1,24 @@
+begin;
+
+with rows as (
+insert into ggircs_portal.emission_category(id, swrs_emission_category, display_name)
+overriding system value
+values
+(1, 'BC_ScheduleB_GeneralStationaryCombustionEmissions', 'Stationary Fuel Combustion'),
+(2, 'BC_ScheduleB_VentingEmissions', 'Venting'),
+(3, 'BC_ScheduleB_FlaringEmissions', 'Flaring'),
+(4, 'BC_ScheduleB_FugitiveEmissions', 'Fugitive/Other'),
+(5, 'BC_ScheduleB_OnSiteTransportationEmissions', 'On-Site Transportation'),
+(6, 'BC_ScheduleB_WasteEmissions', 'Waste'),
+(7, 'BC_ScheduleB_WastewaterEmissions', 'Wastewater'),
+(8, 'BC_ScheduleB_IndustrialProcessEmissions', 'Industrial Process')
```

Comment:
Industrial Process should be the second one in the list, as per Robert's comment here: https://ciip-portal-wksv3k-test-feedback.pathfinder.gov.bc.ca/posts/8/no-section-for-industrial-process-emissions